modulus/models/diffusion/song_unet.py

```diff
@@ -610,3 +610,281 @@ def _get_positional_embedding(self):
         else:
             raise ValueError("Gridtype not supported.")
         return grid
+
+
+class SongUNetPosLtEmbd(SongUNet):
```

Comment:
Could you add a sentence explaining how this is different from `SongUNetPosEmbd`?

Reply:
Description will be added: This model is adapated from SongUNetPosEmbd, with the incoporatation of lead-time aware embedding for the GEFS-HRRR model. The lead-time embedding is activated by setting the lead_time_channels and lead_time_steps parameters.